Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2766,3 +2766,10 @@ extension BrowserViewController {
                                      extras: [TelemetryWrapper.EventExtraKey.isInvertColorsEnabled.rawValue: UIAccessibility.isInvertColorsEnabled.description])
     }
 }
+
+extension BrowserViewController: WhatsNewViewDelegate {
+    
+    func whatsNewViewDidShow(_ viewController: WhatsNewViewController) {
```

Comment:
Sorry, might have missed something on the description, is this planned on being used at some point?

Reply:
It’s indeed being planned to be used! I think I will likely go after a similar implementation of the current Default Browser. Bit tricky but lastly the way to go in my opinion.
Removed the `Draft` too early this time 😅